src/KafkaFlow.Serializer.JsonCore/JsonCoreSerializer.cs

```diff
@@ -1,28 +1,28 @@
-﻿namespace KafkaFlow.Serializer.Json
+﻿namespace KafkaFlow.Serializer
 {
     using System;
     using System.Text.Json;
 
     /// <summary>
     /// A message serializer using System.Text.Json library
     /// </summary>
-    public class JsonMessageSerializer : IMessageSerializer
+    public class JsonCoreSerializer : IMessageSerializer
```

Comment:
Before we had `XYXMessageSerializer` extending `IMessageSerializer`.

Now we have `XYXSerializer`, so maybe we should rename `IMessageSerializer` to `ISerializer`.

What do you think?

Reply:
In the past, I called IMessageSerializer to not create a conflict with the ISerializer of the .NET framework and other things that has this interface defined. I do prefer the ISerializer only too. What do you think?

Reply:
I agree. Let's change to ISerializer so.

Reply:
It is better to do it when we solve [this ](https://github.com/Farfetch/kafka-flow/issues/142) issue.